refinery/tool_manager/models.py

```diff
@@ -1153,6 +1153,20 @@ def get_galaxy_import_tasks(self):
             ) for file_store_item_uuid in self.get_input_file_uuid_list()
         ]
 
+    def get_refinery_import_tasks(self):
```

Comment:
This method should be a part of the `Analysis` model. Then you won't need to obtain a `Tool` object in `_refinery_file_import()`.
Also, technically, this method returns [task signatures](http://docs.celeryproject.org/en/3.1/userguide/canvas.html#signatures), so it should probably be called something like `get_refinery_import_task_signatures` to avoid ambiguity.